lib/hashes.md

```diff
@@ -5,40 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A Hash is a collection of data elements that are indexed by a key that is another data element.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+A Hash would be used to organize data into pairs, such as names and addresses or cities and states.
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Both are collections of data elements, but Arrays are indexed by sequential integers, while Hashes are indexed by other data elements.
 
 # How do you retrieve a particular value from a Hash?
 
-Replace me with your response...
+By using the data element associated with that value as the index key.
```

Comment:
I think I'm probably not using the correct terminology.  By "data element associated with that value" I just meant whatever you're using for the key.  In your hash example above, you would use hash["key1"] to retrieve "value1", hash["key2"] to retrieve "value2" etc, but the key and value can be any kind of Ruby object, as you said.